Vagrantfile

```diff
@@ -35,7 +35,7 @@ Vagrant.configure(2) do |config|
   # config.vm.network "forwarded_port", guest: 80, host: 8082
   config.vm.network "forwarded_port", guest: 5984, host: 5984, auto_correct: true
   config.vm.network "forwarded_port", guest: 5986, host: 5986, auto_correct: true
-  config.vm.network "forwarded_port", guest: 9000, host: 9000, auto_correct: true
+  config.vm.network "forwarded_port", guest: 3000, host: 3000, auto_correct: true
```

Comment:
The master branch still serves the app on port 4200, so you will not be able to access the app on the host machine with this port forwarding.